### DISCUSSION TITLE
兼容性问题？开启清爽首页后首页上的视频封面会快速频闪

### DISCUSSION THREAD

**Discussion by LabDogX** [the1812/Bilibili-Evolved#3075] [提问 / Q&A] [answered]
**对应功能**
清爽首页
Commit: https://github.com/the1812/Bilibili-Evolved/commit/b1ca632fb5f473f9c02620ef327cedb3dffe88a1
内部名称: freshHome

**问题描述**
开启清爽首页后首页上的视频封面会快速频闪，由正常有封面切换到空白，再切换回来，频率非常高，大概1s能够切换两三次（见附录截图）并且只要开启该功能后会持续进行频闪。尝试清除B站的cookies也无法解决该问题。
此外朋友的电脑上开启该功能不会频闪，我们两个人都是新下载的该功能，只有我开启之后会出现该问题，他只下载了脚本的部分功能，我下载了脚本几乎全部的功能，但是并没有全部开启

**脚本版本**
v2.1.5-preview

**浏览器版本**
Chrome 版本 98.0.4758.102（正式版本） （64 位）

**播放器版本**
2.75.3-rc.3220.0

**播放策略**
AVC

**错误信息**
Failed to load resource: the server responded with a status of 403 ()

**附加截图**
<img width="1280" alt="Snipaste_2022-03-02_16-09-49" src="https://user-images.githubusercontent.com/38375576/156323936-87e96fde-4aa7-4719-a595-a7d99654e069.png">
<img width="1280" alt="Snipaste_2022-03-02_16-09-58" src="https://user-images.githubusercontent.com/38375576/156323961-aecf84bc-3c37-4d70-99be-b0b549b84709.png">
<img width="416" alt="Snipaste_2022-03-02_16-21-06" src="https://user-images.githubusercontent.com/38375576/156323981-7cd346f3-ddd6-44a6-920e-38122c508305.png">
<img width="416" alt="Snipaste_2022-03-02_16-31-26" src="https://user-images.githubusercontent.com/38375576/156324462-13a04a2f-9562-4884-bf81-035d78f623b2.png">

**Comment by the1812** [comment, marked as answer]
重复 #3074

**Reply by LabDogX** [reply]
解决了，确实是adblock，搜索清爽首页没搜到那个issue我以为没人问……
